contribs/dvrp/src/main/java/org/matsim/contrib/dvrp/passenger/AbstractPassengerRequestEvent.java

```diff
@@ -35,19 +37,20 @@
  *
  * @author Michal Maciejewski (michalm)
  */
-public abstract class AbstractPassengerRequestEvent extends Event implements HasPersonId {
+public abstract class AbstractPassengerRequestEvent extends Event {
```

Comment:
@michalmac since all AbstractPassengerRequestEvents can now be about multiple persons (submitted, scheduled, rejected) we can no longer implement the HasPersonId interface which would only return one person id
@michalmac